test/testutil.c

```diff
@@ -34,6 +34,9 @@ typedef struct test_info {
 
 static TEST_INFO all_tests[1024];
 static int num_tests = 0;
+static BIO *b_stderr = NULL;
+static BIO *b_stdout = NULL;
```

Comment:
In apps and older test programs using BIO, the most usual names are `bio_out` and `bio_err`. Would you mind using those names here as well?